### PR TITLE
Update code after rebase

### PR DESCRIPTION
**Pull request by jcoyne**
Mostly addresses RIIIF issues.

### CODE REVIEW

File: spec/controllers/hyrax/generic_works_controller_spec.rb [hunk 1]
```diff
@@ -486,14 +486,19 @@
...
     let(:manifest_factory) { double(to_h: { test: 'manifest' }) }
 
+    let(:file_set) do
+      create_for_repository(:file_set,
+                            user: user,
+                            content: file)
+    end
+    let(:file) { fixture_file_upload('/world.png', 'image/png') }
+    let(:work) do
+      create_for_repository(:work, user: user, member_ids: [file_set.id])
```
Comment (kdid) [inline spec/controllers/hyrax/generic_works_controller_spec.rb]: Does this need to be a public work in order for the "produces a manifest" test to pass? I'm getting an Authorization Required result.

File: spec/presenters/hyrax/file_set_presenter_spec.rb [hunk 2]
```diff
@@ -285,19 +284,25 @@
...
+          end
+
+          let(:file) { fixture_file_upload('/hyrax_generic_stub.txt', 'plain/text') }
 
           it { is_expected.to be_nil }
         end
 
         context "when the file is an image" do
-          let(:file_path) { File.open(fixture_path + '/world.png') }
+          let(:file) { fixture_file_upload('/world.png', 'image/png') }
```
Comment (kdid) [inline spec/presenters/hyrax/file_set_presenter_spec.rb]: I'm seeing these errors ```NoMethodError:
        undefined method `fixture_file_upload'```

File: spec/requests/riiif_spec.rb [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
-  before do
-    Hydra::Works::AddFileToFileSet.call(file_set,
-                                        File.open(fixture_path + '/world.png'),
-                                        :original_file)
+  let(:file_set) do
+    create_for_repository(:file_set,
+                          user: user,
+                          content: file)
+  end
+  let(:file) { fixture_file_upload('/world.png', 'image/png') }
```
Comment (kdid) [inline spec/requests/riiif_spec.rb]: I'm getting `RuntimeError: /world.png file does not exist` here. Not sure if this has something to do with including `ActionDispatch::TestProcess`.

Comment (jcoyne) [reply spec/requests/riiif_spec.rb]: I'm not seeing this error on my computer.

File: spec/controllers/hyrax/generic_works_controller_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/controllers/hyrax/generic_works_controller_spec.rb]: I don't know, I just changed it such that it wasn't trying to attach files using the Hydra::Works services.

File: spec/presenters/hyrax/file_set_presenter_spec.rb (hunk 2 above)
Comment (jcoyne) [reply spec/presenters/hyrax/file_set_presenter_spec.rb]: I do not see this.  How are you running it? I'm doing `bundle exec rspec spec/presenters/hyrax/file_set_presenter_spec.rb`

**Pull request merged**